doc/adoc/user_guide.adoc

```diff
@@ -284,19 +301,25 @@ independent initrd can be created by setting
 build_host_independent_initrd: true|false
 
 == Run the Migration
+Migration can be triggered either via run_migration or via reboot.
+
+Option 1 - Running Migration via run_migration::
 After the install of the `SLES15-Migration` package, start the migration
-by calling the following command:
+process by calling the following command:
 
 [listing]
 tux > sudo run_migration
 
+Option 2 - Running Migration via reboot::
 [NOTE]
-If the `suse-migration-sle15-activation` package was installed,
-start the migration by a reboot of the system as follows:
+If using the `reboot` method and install option was used,
```

Comment:
this still seems odd "and install option was use"

Reply:
good catch, changed it to
```
If using the `reboot` method to start migration, reboot the system:
```